### ISSUE TITLE
Warning after Login

### ISSUE THREAD

**Issue by Buckrogers81** [Necrobot-Private/NecroBot#1558]
I have been Using the latest Version of the Bot. Yesterday  I logged on with my Mobile-Device and got the following Message. 

It means in Short: You have been using 3rd Party Software in a non apropiate way!!! Stop doing it

How safe is it to use the Bot?
![pgmeldung](https://cloud.githubusercontent.com/assets/21248749/25324205/c2ae8664-28c4-11e7-9c77-e0966da3c682.PNG)

A friend is using Necrobot and is getting the same Warning.

**Comment by mboeder** [comment]
same here

**Comment by Zamuzo** [comment]
same issue

**Comment by cjhart** [comment]
3 account running Necrobot and I get this now on all 3 when I log in using my device.

**Comment by eversonpiza** [comment]
same here in my 3 accounts

**Comment by cjhart** [comment]
Lord Haji, why did you close this?
Is it just coincidence that we all are getting this and all use Necrobot?

**Comment by jjskuld** [comment]
Looks like Niantic has added new limits for number of pokemon caught and pokestops spun.  The limits are now dynamically changing according to the data that we have.  Based on users in the PogoDev forum, here's how it seems to work:

1) New accounts have limits of 2100 pokestops and 1500 pokemon before you get softbanned.
2) If exceed half of the limit (e.g. around 750), then the next day the limits seem to be decreased to 500.
3) And then if you hit the limit again that day, then the limits are decreased to 200 or 300 the following day.

TLDR - Bottom line: Set your limits per day to 500 pokemons and 700 pokestops to avoid getting softbanned.  We'll be changing the defaults in the bot and decreasing the limits in the next release.

I'll re-open this for the benefit of users getting this warning.

**Comment by zinzin64** [comment]
may thanks for explain and issue!

**Comment by jjskuld** [comment]
Softban limits are now updated in v159.

**Comment by cjhart** [comment]
That's great, thanks for the update.
I have stopped for a few days, hoping the ban message will go, been 3 or 4 days now.

Question:
Does anyone have any idea how long the ban warning message lasts?

Thanks for the hard work in developing this software.

**Comment by Buckrogers81** [comment]
The Message shows up for 2 Weeks!

I set my Limits to 350 per Day and still get the message even when I let the Bot run for just 3 hours a day. There must be more that just setting the limit to 500!

**Comment by blackms** [comment]
What in “API Implementation is not correct” is hard to understand?

Don’t bot.

**Comment by Buckrogers81** [comment]
I was Just anwsering a Question! So relax! I dont bot at the moment!

**Comment by blackuGT** [comment]
My account was banned week after this information
